TMS.Net07.Homework.DaysOfWeek/Master/Program.cs

```diff
@@ -0,0 +1,171 @@
+                    break;
+                case 6:
+                    amountThisYear = 181 + day;
+                    if (year % 4 == 0)
+                        amountThisYear++;
+                    break;
+                case 7:
+                    amountThisYear = 212 + day;
+                    if (year % 4 == 0)
+                        amountThisYear++;
+                    break;
+                case 8:
+                    amountThisYear = 243 + day;
+                    if (year % 4 == 0)
+                        amountThisYear++;
+                    break;
+                case 9:
+                    amountThisYear = 273 + day;
+                    if (year % 4 == 0)
+                        amountThisYear++;
+                    break;
+                case 10:
+                    amountThisYear = 304 + day;
+                    if (year % 4 == 0)
+                        amountThisYear++;
+                    break;
+                case 11:
+                    amountThisYear = 334 + day;
+                    if (year % 4 == 0)
+                        amountThisYear++;
+                    break;
+            }
+            return (amountPastYears + amountThisYear);
+        }
+        static string CalculateDayOfWeek(int day, int month, int year)
+        {
+            switch (CalculateAmountOfDays(day, month, year) % 7)
+            {
+                case 1:
+                    return "Monday";
+                case 2:
+                    return "Tuesday";
+                case 3:
+                    return "Wednesday";
+                case 4:
+                    return "Thursday";
+                case 5:
+                    return "Friday";
+                case 6:
+                    return "Saturday";
+                case 0:
+                    return "Sunday";
+                default:
+                    break;
+            }
+            return "Error";
+        }
+        static void Main(string[] args)
+        {
+            string dateFormat = @"[0-3]\d\.[0-1]\d\.\d{4}"; //doesn't check 100% adequacy
+            while (true)
+            {
+                Console.WriteLine("Enter date in format DD.MM.YYYY:");
+                string input = Console.ReadLine();
+                if (input.ToLower() == "exit")
+                {
+                    Console.WriteLine("\nThis command finish the program. Good bye!");
+                    break;
+                }
+                if (Regex.IsMatch(input, dateFormat))
+                {
+                    //I can use Parse instead of TryParse because of regex above
+                    int day = int.Parse(input.Substring(0, 2));
+                    int month = int.Parse(input.Substring(3, 2));
+                    int year = int.Parse(input.Substring(6));
+                    if (CheckDate(day, month, year))
+                        Console.WriteLine("\nIt's " + CalculateDayOfWeek(day, month, year) + ". Enter another date or \"exit\" to exit.\n");
```

Comment:
Suggest to use `{` `}` even for single statements
```suggestion
                    {
                        Console.WriteLine("\nIt's " + CalculateDayOfWeek(day, month, year) + ". Enter another date or \"exit\" to exit.\n");
                    }
```